docs/extinction.rst

```diff
@@ -55,7 +55,7 @@ https://www.aanda.org/articles/aa/pdf/2011/03/aa15537-10.pdf.
 available at https://www.apo.nmsu.edu/arc35m/Instruments/DIS/ (https://www.apo.nmsu.edu/arc35m/Instruments/DIS/images/apoextinct.dat).
 
 In each case, the extinction is given in magnitudes per airmass and the wavelengths are in Angstroms. Here is an example that
-uses the `AtmosphericExtinction` class to load each model and plots the extinction in magnitudes as well as fractional transmission
+uses the ``AtmosphericExtinction`` class to load each model and plots the extinction in magnitudes as well as fractional transmission
```

Comment:
Why not link to the class API?

Reply:
yeah, i should do that. wanted to at least get it working first.

Reply:
done.